polygon-digital-carbon/schema.graphql

```diff
@@ -441,7 +441,7 @@ type C3RetirementMetadata @entity {
   reason: String
   projectId: String
   projectAddress: Bytes
-  amount: Int
+  amount: String
```

Comment:
should this be a floating point Number instead? I'm worried that using String is going to cause some type issues elsewhere...